### PR TITLE
Add uuid package for Observability

### PR DESCRIPTION
**Pull request by sriteja777**
Jira: https://browserstack.atlassian.net/browse/OB-1679

### CODE REVIEW

**Review by swrdfish — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -43,7 +43,9 @@
     "git-repo-info": "^2.1.1",
     "gitconfiglocal": "^2.1.0",
     "request": "^2.88.2",
-    "strip-ansi": "^6.0.1"
+    "sinon": "^15.2.0",
```
Comment (swrdfish) [inline package.json:46]: `sinon` should be a devDependency

File: test/src/testObservability.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  beforeEach(() => {
+    sandbox = sinon.createSandbox();
+    testObservability = new TestObservability();
+  });
+
+  afterEach(() => {
+    sandbox.restore();
+  });
+
+  describe('processTestRunData', function () {
```
Comment (swrdfish) [inline test/src/testObservability.js:16]: don't use nested describe blocks, we use a single describe block per file by convention.

File: package.json (hunk 1 above)
Comment (sriteja777) [reply package.json]: missed, moved

File: test/src/testObservability.js (hunk 2 above)
Comment (sriteja777) [reply test/src/testObservability.js:16]: Any reason why? I feel nested describe blocks are better at maintaining proper context and helps in reducing redundancy code

Comment (swrdfish) [reply test/src/testObservability.js:16]: It helps in grouping, does not help in reducing redundant code. We use separate files to do the grouping, keeps things simple.

**Review by swrdfish — CHANGES_REQUESTED** [review]

File: test/src/testObservability.js (hunk 2 above)
Comment (sriteja777) [reply test/src/testObservability.js]: Have removed nested block and moved it to a different file

Comment (sriteja777) [reply test/src/testObservability.js]: It does help in reducing redundant code. Example in this case, the variables, stubs and other necessary things for TestObservability class can be defined in a parent describe block, thereby decreasing code to mock it in every function tests.

I do agree having separate files will be good to keep things simple, but allowing multiple describe blocks helps us to write tests cases with different contexts and it also helps in readable code

**Pull request merged**